app/src/androidTest/java/com/willowtree/vocable/PhrasesUseCaseTest.kt

```diff
@@ -58,23 +61,100 @@ class PhrasesUseCaseTest {
                 sortOrder = 0
             )
         )
+        dateProvider.time = 101L
+        presetPhrasesRepository.updatePhraseLastSpokenTime("category_123_0")
```

Comment:
Since the database defaults lastSpokenTime to null and we have a `WHERE ... NOT NULL` somewhere in our query, we only ever return phrases which have ever been spoken once before 

<img width="728" alt="Screenshot 2024-02-02 at 12 54 39 PM" src="https://github.com/willowtreeapps/vocable-android/assets/58408581/bfaa33be-2004-4a36-a396-9f497b22ad0e">

Reply:
Ah, that makes sense!